server/conf/i18n/messages

```diff
@@ -97,6 +97,49 @@ title.submittedPrograms=Submitted
 toast.applicationSaved=Successfully saved application: application ID {0}
 toast.programCompleted=Application was already completed.
 
+#------------------------------------------------------------------------#
+# TRUSTED INTERMEDIARY DASHBOARD PAGE - text when adding a client #
+#------------------------------------------------------------------------#
+
+banner.clientInfoUpdated=Client info has been successfully updated.
+banner.clientAcctDeleted=The client account for "{0}" has been successfully deleted.
+banner.acctDeleteError=This account cannot be deleted.
+
+button.addNewClient=Add new client
+button.backToEditing=Back to editing
+button.cancel=Cancel
+button.deleteAcct=Delete account
+button.keepAcct=Keep account
+button.nextPage=Next
+button.save=Save
+button.search=Search
+button.viewApplications=View applications
+
+content.numberOfAppSubmitted={0} applications submitted
+content.warning=There are no applications submitted for this client, so no applications will be lost.
+
+dialog.deleteConfirmation=Are you sure you want to delete this client's account?
```

Comment:
suggestion (blocking):  For some reason (maybe an escape character?), you need to put two apostrophes here.  "...to delete this client''s account?".  See other examples in the messages file.